docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,98 @@
+            - create a variable (power_validation_status) to track whether the power validation passes and set it to "PASS" - if the power validation fails at any point, this variable will change to give the values at which teh power validation failed: `power_validation_status = "PASS"
+
+            - convert the chiller.capacity_validation_points to a dictionary with the keys being a list of two values - [chilled_water_supply_temperature, condenser_temperature], and values being the capacity_validation_point itself.  Sorting the validation points will allow the calculations that follow to be much simpler.  Start by creating a dict capacity_validation_pts_dict: `capacity_validation_pts_dict = {}`
+            - look at each value in chiller.capacity_validation_points: `for capacity_validation_point in chiller.capacity_validation_points:`
+                - add the point to capacity_validation_pts_dict: `capacity_validation_pts_dict[[capacity_validation_point.chilled_water_supply_temperature, capacity_validation_point.condenser_temperature]] = capacity_validation_point`
+            
+            - do the same conversion with the chiller.power_validation_points and storing the results in power_validation_pts_dict.  However in this case, the values are a list of points because we expect different loads at each point: `power_validation_pts_dict = {}`
+            - look at each value in chiller.power_validation_points: `for power_validation_point in chiller.power_validation_points:`
+                - set the default to a blank list: `power_validation_pts_dict.setdefault([[power_validation_point.chilled_water_supply_temperature, power_validation_point.condenser_temperature]], [])`
+                - append the power validation point to the list at this given: `power_validation_pts_dict[[power_validation_point.chilled_water_supply_temperature, power_validation_point.condenser_temperature]].append(power_validation_point)`
+
+            - create a dictionary of the expected capacities for use in the power validation check: `given_capacities = {}`
+            - the following lines or logic do the capacity validation check:
+            - loop through the expected_cwt_temps: `for cwt in expected_cwt_temps:`
+                - loop through each expected_ect_temps: `for ect in expected_ect_temps:`
+                    - look for the capacity validation point in capacity_validation_pts_dict: `if capacity_validation_pts_dict[[cwt, ect]]:`
+                        - calculate the expected capacity by multiplying the result of the formula by the rated_capacity: `expected_capacity = (cap_f_t_coefficients[0] + cap_f_t_coefficients[1] * cwt + cap_f_t_coefficients[2] * cwt^2 + cap_f_t_coefficients[3] * ect + cap_f_t_coefficients[4] * ect^2 + cap_f_t_coefficients[5] * cwt * ect) * rated_capacity`
+                        - get the given capacity: `given_capacity = capacity_validation_pts_dict[[cwt, ect]].result`
```

Comment:
not following the .result. I think maybe i am missing something but not seeing whyyou would need the .result or how it would know what that means.

Reply:
this is related to [your comment above](https://github.com/pnnl/ruleset-checking-tool/pull/1520/files#r1750266631) - the values were validation points, so we needed to call `.result` to get the output.  I've changed the dictionary and this line, so the values in `capacity_validation_pts_dict` are simply results.